.github/workflows/main.yml

```diff
@@ -7,15 +7,22 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 14.x
+          node-version: 18.x
       - run: |
           yarn && yarn lint && yarn test
-          cd react-native-in-app-payments-quickstart && yarn && yarn lint && cd .. 
+          cd react-native-in-app-payments-quickstart && yarn && cd .. 
 
   build-android:
     needs: install-and-test
     runs-on: ubuntu-latest
-    steps:      
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
```

Comment:
Slightly surprised this comes in as a "upgrade version" PR... is it related to the version bump, or more a general improvement?  

(Clearly we do need a good Java, so that's fine, but why didn't we have this-or-something-like-it previously?  Given we didn't, why do we need this now?)

Reply:
For the new version of React-Native i.e 0.74.1 we need to use JDK 17 or else it was throwing error.